desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/PictureDisplayViewModel.kt

```diff
@@ -29,11 +33,25 @@ class PictureDisplayViewModel(
     init {
         pictures.onEach {
             if (totalPictures.value == 0) setPicture(it)
+            val td = Toastnotification()
+            td.displayNotification()
             totalPictures.value = pictures.replayCache.size
         }.launchIn(viewModelScope)
     }
 
 
+    class Toastnotification {
+        fun displayNotification() {
+            val tray = SystemTray.getSystemTray()
```

Comment:
If you already made a class to store data, why are you setting `tray`, `image` or `trayIcon` on every function call.